build-linux.sh

```diff
@@ -3,11 +3,25 @@ set -e
 
 export PATH="$HOME/miniconda/bin:$PATH"
 
+# add flat buffers
+cd ..
+git clone git@github.com:google/flatbuffers.git
+cd ./flatbuffers
+git checkout 925c1d77fcc72636924c3c13428a34180c30f96f  # release commit
+cmake 
+make
+sudo make install
+sudo cp ./flatc /usr/local/bin/
+cd ..  # to ../
+cd ./vovowpal_wabbit
```

Comment:
merge to one command 'cd ../vowpal_wabbit'. Also, vowpal_wabbit is spelled incorrectly.